liquibase-standard/src/main/java/liquibase/snapshot/DatabaseSnapshot.java

```diff
@@ -690,8 +690,6 @@ public ParsedNode serialize() {
 
     /**
      * Used to get and store misc data that should be scoped to the snapshot. Helpful for caching snapshot results.
-     *
-     * @deprecated Will be removed with 4.0
```

Comment:
The undeprecation at DatabaseSnapshot.java is not related here, just using the PR to merge it too .